designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceSpec.scala

```diff
@@ -266,6 +272,41 @@ class DeploymentServiceSpec
     }
   }
 
+  test { "should return error when trying to deploy without comment when comment is required" } {
```

Comment:
While we are moving validation from route to insides of `DeploymentService` it is valid to keep validation tests here. Is it the reason why you removed test from ManagementResourcesSpeec? If it is so, consider moving all validations tests here.